src/modelldcatnotordf/modelldcatno.py

```diff
@@ -47,6 +59,11 @@ class InformationModel(Resource):
         "_homepage",
         "_contactpoints",
         "_locations",
+        "_modified",
```

Comment:
Det er jo mange flere felter enn dette som er optional for InformationModel, hva bestemmer hvilke du legger til her?